### DISCUSSION TITLE
Feedback: listen()

### DISCUSSION THREAD

**Discussion by nnnnat** [mswjs/mswjs.io#412] [Q&A] [answered]
- [Open page](https://mswjs.io/docs/api/setup-server/listen)

## Feedback

Document the ListenOptions on this page. I was also unable to find this type in the codebase so I do not know what options can be passed in here

**Comment by kettanaito** [comment, marked as answer]
Hi, @nnnnat. Thanks for proposing this.

We have documented the options accepted by the `server.listen()` method [here](https://mswjs.io/docs/api/setup-server/listen#options), right on the `.listen()` page. 

Were these not what you expected?

**Reply by nnnnat** [reply]
:facepalm: my reading comprehension must have been low yesterday. Yes this is exactly what I was expecting lol. Keep up the great work and I'll make sure to read twice next time.

**Reply by kettanaito** [reply]
No worries! Grateful for your feedback regardless.
